xmodule/course_block.py

```diff
@@ -57,6 +58,17 @@
 COURSE_VIDEO_SHARING_PER_VIDEO = 'per-video'
 COURSE_VIDEO_SHARING_ALL_VIDEOS = 'all-on'
 COURSE_VIDEO_SHARING_NONE = 'all-off'
+# .. toggle_name: FEATURES['CREATE_COURSE_WITH_DEFAULT_ENROLLMENT_START_DATE']
+# .. toggle_implementation: SettingDictToggle
+# .. toggle_default: False
+# .. toggle_description: When enabled the newly created courses will have the enrollment_start_date
+#   set to DEFAULT_START_DATE.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2023-06-22
+# .. toggle_target_removal_date: None
```

Comment:
You should drop this annotation altogether.
```suggestion
```

In the future, another annotation will make it clear that this is a permanent toggle, but for now, you could something like: "This is intended to be a permanent option."

Reply:
Replaced with the proposed description